permissions/plugin-opsgenie.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "opsgenie"
+paths:
+- "com/opsgenie/integration/opsgenie"
+- "org/jenkins-ci/plugins/opsgenie"
```

Comment:
There were two groupId in pom.xml so I could not be sure which one is the right one.